pkg/importer/spanner/test-spanner.arrai

```diff
@@ -0,0 +1 @@
+(relations: {(attributes: {(array: {}, length: 0, name: 'Balance', nullable: {}, options: {}, type: 'int64'), (array: {}, length: 6, name: 'BSB', nullable: {}, options: {}, type: 'string'), (array: {}, length: 23, name: 'AccountNum', nullable: {}, options: {}, type: 'string')}, cluster: {}, foreign_keys: {}, name: 'Account', primary_key: {'AccountNum'}), (attributes: {(array: {}, length: 10, name: 'AddressPostCode', nullable: {}, options: 'allow_commit_timestamp=true', type: 'string'), (array: {}, length: 23, name: 'AccountNum', nullable: {}, options: {}, type: 'string'), (array: {}, length: 100, name: 'AddressLine1', nullable: true, options: {}, type: 'string'), (array: {}, length: 100, name: 'AddressLine2', nullable: true, options: {}, type: 'string'), (array: {}, length: 100, name: 'AddressLine3', nullable: true, options: {}, type: 'string')}, cluster: [(interleaved_in: 'Account', on_delete: 'CASCADE')], foreign_keys: {}, name: 'AccountAddress', primary_key: {'AccountNum', 'AddressPostCode'}), (attributes: {(array: {}, length: 10, name: 'LegalRole', nullable: {}, options: {}, type: 'string'), (array: {}, length: 23, name: 'AccountNum', nullable: {}, options: {}, type: 'string'), (array: {}, length: 36, name: 'CustomerID', nullable: {}, options: {}, type: 'string'), (array: true, length: 10, name: 'Permissions', nullable: true, options: {}, type: 'string')}, cluster: {}, foreign_keys: {(constraint_name: 'FK_AccountNum', foreign_keys: {(attribute: 'AccountNum', reference_attribute: 'AccountNum', reference_table: 'Account')}), (constraint_name: 'FK_CustomerID', foreign_keys: {(attribute: 'CustomerID', reference_attribute: 'CustomerID', reference_table: 'Customer')})}, name: 'CustomerHasAccount', primary_key: {'AccountNum', 'CustomerID'}), (attributes: {(array: {}, length: 10, name: 'Mobile', nullable: true, options: {}, type: 'string'), (array: {}, length: 36, name: 'CustomerID', nullable: {}, options: {}, type: 'string'), (array: {}, length: 64, name: 'FirstName', nullable: {}, options: {}, type: 'string'), (array: {}, length: 64, name: 'LastName', nullable: {}, options: {}, type: 'string'), (array: {}, length: 256, name: 'Email', nullable: true, options: {}, type: 'string')}, cluster: {}, foreign_keys: {}, name: 'Customer', primary_key: {'CustomerID'}), (interleaved_table: {}, key_part: 'AccountNum DESC', name: 'AccountsByNum', nullfiltered: {}, storing_col: {}, table_name: 'Account', unique: {}), (interleaved_table: 'Customer', key_part: 'Email ASC', name: 'CustomerByEmail', nullfiltered: 'NULL_FILTERED', storing_col: ['Email', 'Mobile'], table_name: 'Customer', unique: 'UNIQUE'), (name: 'CustomerAccounts')})
```

Comment:
Nit: If this is what the output is expected to look like (i.e. it's a golden file), please call it `*.golden.arrai` so it's apparent, and ideally generate it with `//fmt.pretty` so it's more human readable. The comparison should be done in arr.ai so the whitespace shouldn't matter.

Reply:
I don't intend to make it a golden file, instead there will be a golden file created for sysl output generated from this file. This file may not even exist once we get the sysl output generated.

Reply:
I don't intend to make it a golden file, instead there will be a golden file created for sysl output generated from this file. This file may not even exist once we get the sysl output generated.